spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'bank_account'
+
+describe Bank::Account do
+  let(:account) {Bank::Account.new("check_it", 1000)}
+
+  describe "attributes" do
+    it "responds to :balance" do
+      expect(account).to respond_to :balance
+    end
+
+    it "responds to :id" do
+      expect(account).to respond_to :id
+    end
```

Comment:
This test and the one before it are great tests to start with, but you don't really need them. Your tests on lines 15 and 19 implicitly cover these tests.